crucible/src/Lang/Crucible/Simulator/OverrideSim.hs

```diff
@@ -246,6 +247,12 @@ bindFnHandle ::
 bindFnHandle h s =
   stateContext . functionBindings %= FnBindings . insertHandleMap h s . fnBindings
 
+-- | Bind a CFG to its handle.
+--
+-- Computes postdominator information.
+bindCFG :: CFG ext blocks args ret -> OverrideSim p sym ext rtp a r ()
+bindCFG c = bindFnHandle (cfgHandle c) (UseCFG c (postdomInfo c))
```

Comment:
Perhaps we should also have `bindOverride` (which leverages `UseOverride` instead of `UseCFG`) for symmetry.

Reply:
Fair enough. I have no objections to leaving it out.